plugins/rest/rest_handlers.go

```diff
@@ -265,32 +265,36 @@ func (plugin *Plugin) commandHandler(formatter *render.Render) http.HandlerFunc
 
 		body, err := ioutil.ReadAll(req.Body)
 		if err != nil {
-			plugin.Log.Error("Failed to parse request body.")
-			formatter.JSON(w, http.StatusBadRequest, err)
+			errMsg := fmt.Sprintf("400 Bad request: failed to parse request body: %v\n", err)
```

Comment:
There is no point printing status code definition here if it's already always included in the HTTP response.

Reply:
Sure you have, but unless you use verbose or some rest api client, you will not see it. I think it may be handy to have it directly in the message, and also in the log where you can easily identify it during debugging.